blocked-edges/4.13.34-NetPolicyTimeoutsHostNetworkedPodTraffic.yaml

```diff
@@ -0,0 +1,23 @@
+to: 4.13.34
```

Comment:
actually, looks like [4.13.34][1] includes [OCPBUGS-29301](https://issues.redhat.com/browse/OCPBUGS-29301), with a fix for this issue.  So I'd expect a `fixedIn: 4.13.34` in `4.13.33-NetPolicyTimeoutsHostNetworkedPodTraffic.yaml`, and not this risk-extension to 4.13.34.

[1]: https://amd64.ocp.releases.ci.openshift.org/releasestream/4-stable/release/4.13.34

Reply:
Right. Going to update the PR